openmetadata-service/src/main/java/org/openmetadata/service/governance/workflows/WorkflowHandler.java

```diff
@@ -40,6 +40,10 @@ private WorkflowHandler(OpenMetadataApplicationConfig config) {
     ProcessEngineConfiguration processEngineConfiguration =
         new StandaloneProcessEngineConfiguration()
             .setAsyncExecutorActivate(true)
+            .setAsyncExecutorCorePoolSize(50)
```

Comment:
should these be configurable?

Reply:
I think they should, but we can iterate on that after having a decent default for production load.
I was thinking about having a `Workflows Setting` kind of thing within the OpenMetadata settings.
What do you think?